src/js/components/DataTable/doc.js

```diff
@@ -329,6 +329,11 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: '<FormDown />',
   },
+  'dataTable.icons.sortable': {
+    description: 'The icon indicating a column can be sorted.',
```

Comment:
I'm a little confused about whether the icon will be placed in addition to the up/down icon.
Edit: after looking on storybook, we should add a comment that this is an interim icon until the sort action will take place.

Reply:
The icon is intended as an indicator that the column could be sorted. So, it's not exactly "interim". The user might never sort, and that's just fine.

Reply:
```suggestion
    description: 'The icon indicating a column can be sorted. The icon will be replaced by the ascending / descending sort icons once clicked upon.',
```

Reply:
But, then I think we should also say something about it being restored when another column is sorted. We don't currently describe that level of detail for `dataTable.icons.ascending`, where we just say "The ascending icon.". Are you thinking we should document when the icons are used and then removed for all of them?

Reply:
>But, then I think we should also say something about it being restored when another column is sorted

I agree. This level of detail might end up confusing.